caddyconfig/httpcaddyfile/options.go

```diff
@@ -52,6 +52,7 @@ func init() {
 	RegisterGlobalOption("ocsp_stapling", parseOCSPStaplingOptions)
 	RegisterGlobalOption("log", parseLogOptions)
 	RegisterGlobalOption("preferred_chains", parseOptPreferredChains)
+	RegisterGlobalOption("sslkeylogfile", parseOptSingleString)
```

Comment:
We try to avoid the term "SSL" now since it's obsolete. Prefer "TLS" instead.

I'm not super sure a global option is the best way to do this, but maybe it makes sense in practice. I dunno yet. Will have to give it some thought.

Reply:
So I sorta agree, but everyone calls it SSLKEYLOGFILE. e.g.

https://firefox-source-docs.mozilla.org/security/nss/legacy/key_log_format/index.html

https://sslkeylog.readthedocs.io/en/latest/index.html

https://everything.curl.dev/usingcurl/tls/sslkeylogfile

I think it would be confusing to call it tlskeylogfile, as everyone else refers to it as SSL key logging (even when logging non-SSL keys). 

We could call it NSSKeyLogFile, but that still is non-standard. And we aren't using the NSS library, so still slightly confusing to users.

Reply:
I think we will need to support more than just writing to a file. There's no reason we need to use that specific name. I'd probably call the option `insecure_tls_key_export` or something.

Reply:
We should still allow logging to a separate file. It's a hassle to have to manually parse out keys to the "proper" format for usage in tools like Wireshark. (Sometimes it's easier, sometimes it's harder... it's a debugging/troubleshooting feature for many users, so requiring them to write a script just to check something quick isn't ideal.)